frontend/app/components/routing/wp-details/wp-details.controller.test.js

```diff
@@ -28,8 +28,9 @@
 
 /*jshint expr: true*/
 
-describe('WorkPackageDetailsController', function() {
+describe.only('WorkPackageDetailsController', function() {
```

Comment:
The `only` needs to be removed. 

So good that this is the mistake I always make because that way I can spot it quite easily.